client/app/bundles/course/assessment/submission/translations.ts

```diff
@@ -471,12 +477,20 @@ const translations = defineMessages({
       '{stillSomeTimeRemaining, select, true {Once the time is up, \
       the assessment will be automatically finalised.} other {Finalising the submission now!}}',
   },
+  timedAssessmentStartDialogMessage: {
+    id: 'course.assessment.submission.timedExamStartDialogMessage',
+    defaultMessage: 'Click to start the assessment and the timer',
```

Comment:
Can simplify to: `Start the timed assessment`

Reply:
done